Source/MusicAppMediaFix.spoon/init.lua

```diff
@@ -1,6 +1,7 @@
 --- === MusicAppMediaFix ===
 ---
---- Override macOS behaviour and send all media keys (play/prev/next) to Music.app
+--- Override macOS behaviour and send all media keys (play/prev/next) to
+--- the last active of a specified list of apps in MusicAppMediaFix.mediaApps
```

Comment:
I would remove `in MusicAppMediaFix.mediaApps` from this docstring, given the comment below.